test/async-hooks/test-get-connections.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+const common = require('../common');
+const assert = require('assert');
+const net = require('net');
+const server = net.createServer();
+
+/* This test was based on an error raised by Haraka.
```

Comment:
The comment is good 👍 
Style nit: line comments (`//`) are prefered.
And replace `See:` with `Ref:`